tool/teleport/common/teleport.go

```diff
@@ -1096,3 +1138,39 @@ func onJoinOpenSSH(clf config.CommandLineFlags, conf *servicecfg.Config) error {
 	}
 	return nil
 }
+
+func onSELinuxFileContexts(configPath string) error {
+	// Explicitly set the default config path so ReadConfigFile won't
+	// return (nil, nil) if configPath is unset and the default config
+	// path doesn't have a file
+	if configPath == "" {
+		configPath = defaults.ConfigFilePath
+	}
+	cfg, err := config.ReadConfigFile(configPath)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	fileContexts, err := selinux.FileContexts("/opt/teleport/default", cfg.DataDir, configPath)
```

Comment:
Is this the One True Install Directory? Is there a way that we could support custom installation suffixes (as per `teleport-update`), or are we going to support the default suffix only? Can we get the directory name from a const in `lib/autoupdate`?

Reply:
The installation script will only support the default suffix, when `teleport-update` SELinux management is rolled out it will support different suffixes. 

There's a unexported constant for `/opt/teleport` in `lib/autoupdate/agent`, but didn't want to keep expanding the scope and modify that package too